portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,149 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.name {
+  padding-top: 50px;
+  text-transform: uppercase;
+  font-weight: 600;
+  font-size: 48px;
+  font-family: "Avenir Next";
+}
+
+.tag-line {
+  font-size: 24px;
+  font-style: italic;
+  font-family: "Avenir Next";
+}
+
+.icon-list {
+  list-style: none;
+  align-items: center;
+  margin-left: -40px;
+}
+
+.icon {
+  display: inline-block;
+  text-overflow: show;
+  white-space: nowrap;
+  padding-top: 25px;
+  margin-right: 20px;
+}
+
+
+.exp {
+  padding-top: 25px;
+  font-size: 24px;
+  font-family: "Avenir Next";
+  padding-top: 40px;
+  padding-bottom: 20px;
+}
+
+.imgs {
+  height: 75px;
+}
+
+@media only screen and (max-width: 400px) {
+  .imgs {
+    height: 60px;
+  }
+  .name {
+    font-size: 34px;
+  }
+  .descrip {
+    font-size: 16px;
+  }
+  .exp {
+    font-size: 20px;
+  }
+}
+@media only screen and (min-width:400px) and (max-width: 600px) {
+  .name {
+    font-size: 34px;
+  }
+  .descrip {
+    font-size: 16px;
+  }
+  .exp {
+    font-size: 24px;
+  }
+}
+
+@media only screen and (min-width: 600px) and (max-width: 720px) {
+  .descrip {
+    font-size: 18px;
+  }
+}
+
+.exp-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.exp-box {
+  position: relative;
+  padding: 0;
+  width: 125px;
+  height: 125px;
+  display: block;
+  cursor: pointer;
+  overflow: hidden;
+  text-align: center;
+  box-shadow: 0 1px 2px rgba(0,0,0,.12), 0 1px 2px rgba(0,0,0,.24);
+}
+
+.content {
+  font-family: "Avenir Next";
+  opacity: 0;
+  font-size: 14px;
+  position: absolute;
+  top: 0;
+  left: 0;
+  color :white !important;
+  background-color: purple;
+  width: 125px;
+  height: 125px;
+  -webkit-transition: all 400ms ease-out;
+  -moz-transition: all 400ms ease-out;
+  -o-transition: all 400ms ease-out;
+  -ms-transition: all 400ms ease-out;
+  transition: all 400ms ease-out;
```

Comment:
Love to see the transitions too!

Reply:
thanks! after we get a chance to speak about scss, I may look into replacing this all with a mixin, as that seems like a cleaner solution.